Jenkinsfile

```diff
@@ -162,6 +162,28 @@ pipeline {
       }
     }
 
+    stage('L2: Speech to Text WPE') {
```

Comment:
Please parallel this with another test of similar duration

Reply:
BPE models take more memory, but I might be able to put it on GPU 1 alongside speaker embeddings and speech recognition. I'll try.

Reply:
@blisc Done